validator/accounts/v2/wallet_create.go

```diff
@@ -21,23 +21,26 @@ func CreateWallet(cliCtx *cli.Context) error {
 		return err
 	}
 	w, err := NewWallet(cliCtx, keymanagerKind)
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrWalletExists) {
 		return errors.Wrap(err, "could not check if wallet directory exists")
 	}
+	if errors.Is(err, ErrWalletExists) {
+		return ErrWalletExists
```

Comment:
Just returning the wallet exists error here so it doesn't pass through `could not check if wallet directory exists`